R/setPrior.R

```diff
@@ -0,0 +1,105 @@
+
+#priors() method for stanmodule, hazardlink , longmodel and osmodel
+
+# getter method for LongModel:
+setGeneric(name = "prior", def = function(object) standardGeneric("prior"))
+#' get method for longmodel
+setMethod(
+    f = "prior",
+    signature = list(object = "LongModel"),
+    definition = function(object) {
+        object@stan@priors
+    }
+)
+
+#' get method for OsModel
+
+setMethod(
+    f = "prior",
+    signature = list(object = "OsModel"),
+    definition = function(object) {
+        object@stan@priors
+    }
+)
+
+#' get method for Hazard link
+
+setMethod(
+    f = "prior",
+    signature = "HazardLink",
+    definition = function(object) {
+        object@stan@priors
+    }
+)
+
+#' get method for stan module
+
+
+setMethod(
```

Comment:
please move the `StanModule` methods to the file `stan_module.R` - sorry I think I said something else in the last standup, but looking at how the other code is structured that makes more sense to keep the methods with the classes.